lib/plenario_web/router.ex

```diff
@@ -54,6 +54,9 @@ defmodule PlenarioWeb.Router do
     resources "/data-sets", DataSetController
     post "/data-sets/:id/submit-for-approval", DataSetController, :submit_for_approval
     post "/data-sets/:id/ingest-now", DataSetController, :ingest_now
+    get "/data-sets/:id/request-changes", DataSetController, :request_changes
+    post "/data-sets/:id/send-email", DataSetController, :send_email
```

Comment:
Kind of pedantic, but I think a better path for this would be `/data-sets/:id/send-change-request-email`.